Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -231,6 +258,27 @@ open class ChatChannelListVC: _ViewController,
             self?.isPaginatingChannels = false
         }
     }
+    
+    /// Shows the error view.
+    open func showErrorView() {
+        channelListErrorView.isHidden = false
+        
+        UIView.animate(withDuration: 0.5) {
+            self.channelListErrorView.center = .init(x: self.channelListErrorView.center.x, y: self.channelListErrorView.center.y - self.channelListErrorViewHeight)
```

Comment:
Can you explain why we need to do this? Maybe adding a comment?

Reply:
This is used to animate the error view by changing its center. Moved to the error view as suggested in another comment 👍🏻